include/ishikawa.hrl

```diff
@@ -1,3 +1,10 @@
 -type actor() :: binary().
 -type message() :: binary().
 -type timestamp() :: vclock:vclock().
+-type timestampMatrix() :: mclock:mclock().
```

Comment:
`timestamp_matrix` would be more idiomatic: the only thing that's usually CamelCase in Erlang are variables.